src/main/java/core/basesyntax/Truck.java

```diff
@@ -0,0 +1,13 @@
+package core.basesyntax;
+
+public class Truck extends Machine {
+    @Override
+    public void doWork() {
+        System.out.println("Truck is work");
+    }
+
+    @Override
+    public void stopWork() {
+        System.out.println("Truck is stop work");

```

Comment:
Similarly, the message in the `stopWork` method should be more informative. Consider revising it to "Truck has stopped working." to clearly reflect the action.